lib/bridge/IrcHandler.js

```diff
@@ -16,6 +16,12 @@ function IrcHandler(ircBridge) {
     this._roomIdToPrivateMember = {
         // room_id: { user_id: $USER_ID, membership: "join|invite|leave|etc" }
     };
+
+    // Maintain a map of topics so as not to repeatidly send duplicate topics into
+    // a room.
+    this._roomIdToTopic = {
```

Comment:
Oh I see, I thought we were happy with passing topics through on startup, my mistake.